codecov.yml

```
@@ -14,4 +15,11 @@ coverage:
   layout: "reach,diff,flags,tree"
   behavior: default
-  require_changes: yes
\ No newline at end of file
+  require_changes: yes
+
+ignore:
+  - '**/*.pb.go'
+
+fixes:
+  - "github.com/go-fries/fries/(.*)/v3/::\1/"
+  - "github.com/go-fries/fries/v3/::./"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)